tfjs-backend-wasm/src/cc/kernels/BitwiseAnd.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2023 Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ===========================================================================*/
+
+#ifdef __EMSCRIPTEN__
+#include <emscripten.h>
+#endif
+#include <xnnpack.h>
+
+#include <cstddef>
+
+#include "tfjs-backend-wasm/src/cc/binary.h"
+#include "tfjs-backend-wasm/src/cc/util.h"
+
+namespace {
+template <class T>
```

Comment:
`template <typename T>` is preferred go/totw/12#what-do-i-need-to-know-about-templates

(we usually follow the internal language style guides even though it's on github)